resources/views/masjid_profile/edit.blade.php

```diff
@@ -21,4 +31,125 @@
         </div>
     </div>
 </div>
+<div class="modal fade" id="modal-masjid" tabindex="-1" role="dialog" aria-labelledby="modalLabel" aria-hidden="true">
+    <div class="modal-dialog modal-lg" role="document">
+      <div class="modal-content">
+        <div class="modal-header">
+          <h5 class="modal-title" id="modalLabel">{{__('masjid_profile.edit')}}</h5>
+          <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+            <span aria-hidden="true"></span>
+          </button>
+        </div>
+        <div class="modal-body">
+          <div class="img-container">
+              <div class="row justify-content-center text-center">
+                  <div class="col-md-8 justify-content-center text-center">
+                      <img id="image" src="" alt="{{ Setting::get('masjid_name') ?? 'buku masjid'}}">
```

Comment:
Di fitur setting itu kita bisa kasih default value mas: `Setting::get('key', 'default_value');`. Jadi default valuenya ambil nama masjid dari config saja.
```suggestion
                      <img id="image" src="" alt="{{ Setting::get('masjid_name', config('masjid.name')) }}">
```